scripts/policy/misc/capture-loss.zeek

```diff
@@ -56,7 +69,7 @@ event CaptureLoss::take_measurement(last_ts: time, last_acks: count, last_gaps:
 		schedule watch_interval { CaptureLoss::take_measurement(network_time(), 0, 0) };
```

Comment:
```suggestion
		schedule initial_watch_interval { CaptureLoss::take_measurement(network_time(), 0, 0) };
```

To get the intended behavior, I think this needs to change, too.  That is, network-time isn't initially available in `zeek_init()` so any events scheduled within it usually get triggered immediately regardless of the requested interval. At least that's my memory of the pitfall and what still appears to happen when I tested, but can't find if that behavior is documented.

Reply:
Interesting. I think it is documented here: https://docs.zeek.org/en/current/scripts/base/bif/event.bif.zeek.html#id-zeek_init